bmq-sdk/pom.xml

```diff
@@ -21,7 +21,6 @@ limitations under the License. -->
     <version>0.0.10-SNAPSHOT</version>
   </parent>
 
-  <groupId>com.bloomberg.bmq</groupId>
```

Comment:
Could you explain why we are removing this?

Reply:
Static code inspection, we already have a link to the parent in this `.pom` file, here:

https://github.com/bloomberg/blazingmq-sdk-java/blob/0edecbf71c9d4433c1cf2262fdfb572e1590660b/bmq-sdk/pom.xml#L19